spec/filters/json_spec.rb

```diff
@@ -224,6 +226,41 @@
     end
   end
 
+  describe "target" do
```

Comment:
I am hesitant to validate the behaviour of `ECSCompatibilitySupport::TargetCheck` here in a set of unit specs for a plugin that merely _includes_ the target check without overriding any of its behaviour.

Since `ECSCompatibilitySupport::TargetCheck` is well-tested in these scenarios, can we _rely_ on that in any way? Would validating the presence of `ECSCompatibilitySupport::TargetCheck` in the ancestor chain be enough, or could we validate that the decorator's `register` method is called? 

Would it be possible to provide a spec helper or a set of shared examples in the ecs support mixin to reduce the overhead of validating inclusion?

Reply:
thanks - makes sense, was a bit hesitant about the logger asserts - happy to drop them all.

Reply:
> Would it be possible to provide a spec helper or a set of shared examples in the ecs support mixin to reduce the overhead of validating inclusion?

believe it's not necessary given the current scope of the `TargetCheck`